### PR TITLE
2.1.0 - Update Restic to version 0.16.0

### PR DESCRIPTION
**Pull request by dalenewby**
## Changed

* Update Restic to version 0.16.0
* Moved Docker build to CodeShip

### CODE REVIEW

File: aws.env.encrypted [hunk 1]
```diff
@@ -0,0 +1,2 @@
+cloudbees:v1
```
Comment (briskt) [inline aws.env.encrypted]: I don't see any reference to this file. What is in it? We sometimes add a file `aws.env.example` to at least show what variables are needed. For example:

```env
AWS_SECRET_ACCESS_KEY=
AWS_ACCESS_KEY_ID=
AWS_REGION=
```

**Review by briskt — APPROVED** [review]
Looks good. You may have a file that's not needed, though.

**Comment by forevermatt** [comment]
@dalenewby It looks like this GitHub repo doesn't have our normal branch protection rules in place, since it's suggesting deleting the `develop` branch now that you've merged it into `master`. I'll add those.

**Comment by dalenewby** [comment]
Thank you!  It is an older repo.

Dale

**Pull request merged**